curriculum/challenges/english/14-responsive-web-design-22/learn-css-transforms-by-building-a-penguin/619d2f0e9440bc27caee1cec.md

```diff
@@ -9,7 +9,7 @@ dashedName: step-93
 
 Fun fact: Most, if not all, flippers are not naturally rectangles.
 
-Give the `.arm` elements top -left, -right, and bottom-right corners a radius of `30%`, and the bottom-left corner a radius of `120%`.
+Give the `.arm` elements top-left, top-right, and bottom-right corners a radius of `30%`, and the bottom-left corner a radius of `120%`.
```

Comment:
```suggestion
Give the `.arm` elements' top-left, top-right, and bottom-right corners a radius of `30%`, and the bottom-left corner a radius of `120%`.
```

Since we're here, I believe this should be possessive.